middleware/indexer.js

```diff
@@ -119,21 +119,20 @@ const findLikelyTokens = withPgClient(async (ctx) => {
 
     // TODO: Make sure indexer for explorer DB allows for faster way to do it in prod (see also above)
     const mintedWithBridge = `
-        select distinct receiver_account_id from (
-            select convert_from(decode(args->>'args_base64', 'base64'), 'UTF8')::json->>'account_id' as account_id, receiver_account_id
-            from receipts
-            join action_receipt_actions using (receipt_id)
+        select distinct receipt_receiver_account_id as receiver_account_id from (
+            select args->'args_json'->>'account_id' as account_id, receipt_receiver_account_id
```

Comment:
Do you want to have `as receiver_account_id`?

Reply:
not sure what do you mean. it's renamed in outer `select` query.

Reply:
nevermind, I misread the code